catalog/imaging/imaging-transformer-chipping/src/main/java/org/codice/alliance/imaging/chip/transformer/CatalogOutputAdapter.java

```diff
@@ -56,25 +111,420 @@ public BufferedImage getImage(ResourceResponse resourceResponse) throws IOExcept
+    }
+
+    private void setImageCoordinates(int sourceX, int sourceY, int selectWidth, int selectHeight,
+            ImageSegment chipImageSegment, ImageSegment originalImageSegment)
+            throws NitfFormatException {
+        CoordinateConverter coordinateConverter =
+                new CoordinateConverter((int) originalImageSegment.getNumberOfColumns(),
+                        (int) originalImageSegment.getNumberOfRows(),
+                        getFullImageCoordinates(originalImageSegment));
+
+        List<Vector> chipCornerPixels = getChipCornerPixels(sourceX,
+                sourceY,
+                selectWidth,
+                selectHeight);
+
+        List<Vector> chipCornerCoords = coordinateConverter.toLonLat(chipCornerPixels);
+
+        ImageCoordinatePair icp0 = new ImageCoordinatePair();
+        icp0.setFromDMS(formatToDMS(chipCornerCoords.get(0)
+                        .get(1),
+                chipCornerCoords.get(0)
+                        .get(0)));
+
+        ImageCoordinatePair icp1 = new ImageCoordinatePair();
+        icp1.setFromDMS(formatToDMS(chipCornerCoords.get(1)
+                        .get(1),
+                chipCornerCoords.get(1)
+                        .get(0)));
+
+        ImageCoordinatePair icp2 = new ImageCoordinatePair();
+        icp2.setFromDMS(formatToDMS(chipCornerCoords.get(2)
+                        .get(1),
+                chipCornerCoords.get(2)
+                        .get(0)));
+
+        ImageCoordinatePair icp3 = new ImageCoordinatePair();
+        icp3.setFromDMS(formatToDMS(chipCornerCoords.get(3)
+                        .get(1),
+                chipCornerCoords.get(3)
+                        .get(0)));
+
+        chipImageSegment.setImageCoordinates(new ImageCoordinates(new ImageCoordinatePair[] {icp0,
+                icp1, icp2, icp3}));
+        chipImageSegment.setImageCoordinatesRepresentation(ImageCoordinatesRepresentation.GEOGRAPHIC);
+
+        logImageCoordinates("chip image coordinates", chipImageSegment.getImageCoordinates());
+
+    }
+
+    private void logImageCoordinates(String prefix, ImageCoordinates imageCoordinates) {
+        logImageCoordinatePair(prefix, imageCoordinates.getCoordinate00());
+        logImageCoordinatePair(prefix, imageCoordinates.getCoordinate0MaxCol());
+        logImageCoordinatePair(prefix, imageCoordinates.getCoordinateMaxRow0());
+        logImageCoordinatePair(prefix, imageCoordinates.getCoordinateMaxRowMaxCol());
+    }
+
+    private void logImageCoordinatePair(String prefix, ImageCoordinatePair imageCoordinatePair) {
+        LOGGER.info("{} - (lon,lat) {}",
+                prefix,
+                String.format("%.5f,%.5f",
+                        imageCoordinatePair.getLongitude(),
+                        imageCoordinatePair.getLatitude()));
+    }
+
+    private void logVectors(String prefix, List<Vector> vectors) {
+        vectors.forEach(vector -> LOGGER.debug("{} - (lon,lat) {}",
```

Comment:
This log level should probably be at `trace`.  https://codice.atlassian.net/wiki/display/DDF/Log+Levels